lunatrace/bsl/hasura/metadata/remote_schemas.yaml

```diff
@@ -1,7 +1,7 @@
-- name: custom-backend-schema
+- name: lunatrace-custom
```

Comment:
why change this? any remote schema relationships will need to be updated as well if we have them.